charts/spire/README.md

```diff
@@ -105,20 +105,16 @@ Kubernetes: `>=1.21.0-0`
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
 | fullnameOverride | string | `""` |  |
+| global.spire.bundleConfigMap | string | `""` | Override all instances of bundleConfigMap |
+| global.spire.clusterName | string | `""` | Override all instances of clusterName |
+| global.spire.trustDomain | string | `""` | Override all instances of trustDomain |
```

Comment:
What do you think of populating default values here instead of the subchart level?

Reply:
I would expect this to be filled as well.

Reply:
I've done it for two of them. clusterName, and trustDomain. They always need to be the same in all charts.

bundleConfigMap needs to be separately configurable in nestedSpire. With globals recoded the way it is, they take preference to local settings, so you can't set it globally and override it in a single chart.

If we had nested spire pr merged, we'd catch such issues in testing.